### ISSUE TITLE
Tab bar doesn't grow when you resize the window, it only shrinks

### ISSUE THREAD

**Issue by DHowett-MSFT** [microsoft/terminal#3300]
Open a number of tabs. Make the window narrower. Make it wider again.

It looks like this:

![image](https://user-images.githubusercontent.com/14316954/67428516-1f8f6680-f593-11e9-95c2-fb701b239c0c.png)

**Comment by biltongza** [comment]
Adding a new tab makes the tabs pop back to correct size for me.

**Comment by pedrogarcia** [comment]
How come that it's so difficult for UWP apps to ship a fast and decent UI, while other apps mostly always get it right?

**Comment by mdtauk** [comment]
> How come that it's so difficult for UWP apps to ship a fast and decent UI, while other apps mostly always get it right?

It's because this is a Hybrid app, part UWP, part Win32 - that these kinds of resize and rendering bugs are more likely to happen.

**Comment by pedrogarcia** [comment]
> > How come that it's so difficult for UWP apps to ship a fast and decent UI, while other apps mostly always get it right?
> 
> It's because this is a Hybrid app, part UWP, part Win32 - that these kinds of resize and rendering bugs are more likely to happen.

Of course the solution would be to make it entirely UWP. 

Except that I've never seen a UWP app that was actually great to use with keyboard and mouse on a desktop computer. They all feel like a crummy annoying mobile app. 

Btw: the terminal also looks huge (and there is no way to rescale it to sane levels), it's slow (why is maximizing the app window slower and laggier than in other apps?), startup is slower (it does not show up instantly like other apps even on a fast ssd computer).

**Comment by nishp77** [comment]
@DHowett-MSFT there would be an arrow the for resizing it but its in the position like of backslash and then pull it upwards.

**Comment by nishp77** [comment]
Also do I file a bug that is try out add multiple tabs like 10 not on maximized screen but normal window size. Then resize the window and maximize it

**Comment by DHowett-MSFT** [comment]
Please do not file that bug. That is *this* bug. This is that bug.

**Comment by zadjii-msft** [comment]
(this may or may not be the upstream https://github.com/microsoft/microsoft-ui-xaml/issues/1581 - it's unclear at the moment)

**Comment by JushBJJ** [comment]
![4DVgmK4tLY](https://user-images.githubusercontent.com/36951064/69764885-9be21e80-11bd-11ea-9257-0e9c3b82914a.gif)
Is this relevant? Or should this be in a another issue?
Terminal Version: 0.7.3291.0

**Comment by DHowett-MSFT** [comment]
On account of it being literally the same issue, I’d say it belongs here. :)

**Comment by K1ngjulien** [comment]
Do we know by now if this is an issue with ui xaml?

**Comment by zadjii-msft** [comment]
@K1ngjulien with the long holiday weekend, I don't think anyone's had a chance to investigate quite yet

**Comment by K1ngjulien** [comment]
oh sorry I forgot about that, non american here :D

**Comment by zadjii-msft** [comment]
So, I'm happy to announce that this was fixed by the upstream [microsoft/microsoft-ui-xaml#1744](https://github.com/microsoft/microsoft-ui-xaml/pull/1744). It might be a bit before we can ingest the latest build of WinUI, but the fix will be coming soon! Thanks again @teaP!

**Comment by JushBJJ** [comment]
aye

**Comment by rnielikki** [comment]
![ezgif-2-6e3b819e993e](https://user-images.githubusercontent.com/5599742/71712225-07e32280-2e0d-11ea-931f-10743a83f05c.gif)
Not really fixed... the version is 0.7.3451.0 (latest releaase)
